.circleci/config.yml

```diff
@@ -328,6 +328,36 @@ jobs:
               rm -rf build
               cd conda-build
               python linux_matrix_builder.py --ci_test
+  test_conda_build_osx:
+    macos:
+      xcode: 11.3.0
+    steps:
+      - checkout:
+          path: ./habitat-sim
+      - run:
+          name: Brew Install
```

Comment:
No need to install brew here.  conda has cmake, ninja, and ccache and conda build will grab what it needs automatically.  You can just install miniconda directly.